roles/ovirt-external-providers/README.md

```diff
@@ -0,0 +1,74 @@
+# TODO
+
+oVirt External Providers
+========================
+
+The `ovirt-external-providers` role is used set up oVirt external providers like glance.
+
+Requirements
+------------
+
+ * oVirt Python SDK version 4
+ * Ansible version 2.3
+
+Role Variables
+--------------
+
+| Name                  | Default value         |  Description                            |
+|-----------------------|-----------------------|-----------------------------------------|
+| external_providers    | UNDEF                 | List of dictionaries that describe the external provider. |
+
+The `external_providers` list can contain the following parameters:
+
+| Name                              | Default value       | Description                             |
+|-----------------------------------|---------------------|-----------------------------------------|
+| name                              | UNDEF               | Name of the external provider.                     |
+| state                             | present             | State of the external provider.                    |
+| type                              | UNDEF               | Type of the external provider. Values can be os_image, network, os_volume and  foreman. |
+| url                               | UNDEF               | URL where external provider is hosted. |
+| username                          | UNDEF               | Username to be used for login to external provider. Applicable for all types. |
+| password                          | UNDEF               | Password of the user specified in username parameter. Applicable for all types. |
+| tenant                            | UNDEF               | Name of the tenant. |
+| auth_url                          | UNDEF               | Keystone authentication URL of the openstack provider. required: os_image, os_volume and network. |
```

Comment:
So why do you pass it in?

 - roles/ovirt-external-providers/tasks/main.yml

Either remove it or document it.